CHANGELOG.md

```diff
@@ -2,8 +2,16 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## [1.13.0](https://github.com/aws-observability/aws-rum-web/compare/v1.12.0...v1.13.0) (2023-02-23)
+## [1.14.0](https://github.com/aws-observability/aws-rum-web/compare/v1.13.7...v1.14.0) (2023-06-29)
+
+### Features
+
+* Filter unwanted performance events with custom callback. By default, ignore resources with non-http url scheme. ([#419](https://github.com/aws-observability/aws-rum-web/issues/419)) ([42a2ae5](https://github.com/aws-observability/aws-rum-web/commit/42a2ae568b954190db942377df32c7fc5b89686e))
 
+### Bug Fixes
+* Export PageAttributes type. ([#369](https://github.com/aws-observability/aws-rum-web/issues/369)) ([4b78dc1](https://github.com/aws-observability/aws-rum-web/commit/4b78dc14799151422bee07f6263c1aaa098354f6))
```

Comment:
> question: Is this the only bug fix that isn't already in v1.13.7?

#369 was the only bug fix that I found missing in v1.13.7, but that was a manual process. I took the changes from v1.13.0->v1.14.0, and removed the ones duplicated in v1.13.7 and v1.13.0. 

> should we include those fixes in the v1.14.0 change log? I.e., is this CHANGELOG comparing v1.14.0 to v1.13.0 or v1.13.7?

My intuition is to compare 1.14.0 and 1.13.7 as well. However, implementing that comparison is manual and prone to developer mistakes. Moreover, it's unintuitive to readers because patches are not written to the main branch's change log.

Reply:
On quick check, historically it seems we only included immediate changes in minor version bumps and that is what was done here. 

> The question is, should we include those fixes in the v1.14.0 change log? 

Since we keep our patch change log on the release branch, maybe it is a good idea to include the important patches that have come between 1.3 and 1.4 into the main change log for consumers who may not refer to the change logs in the release branch. 

In the future, we could consider adding patch related change log into the main change log to maintain one place with all changes.

Reply:
> My intuition is that the diff should be between v1.14.0 and v1.13.7.

I think my intuition was wrong. The diff should be between v1.14.0 and v1.13.0 for the following reasons.

1. Each branch should have a complete CHANGELOG.
2. The versioning tool should automatically update CHANGELOG.

After a minor version is released, patches are merged to both the release branch and the main branch.

Regarding (1), patches made to a release branch are reflected in the CHANGELOG of the release branch, but not in the CHANGELOG of the main branch.

Regarding (2), I believe the versioning tool  looks for the most recent tag in the current branch, and will build a complete CHANGELOG from the start of the previous release branch.